test/integration/narrow/routes/index.test.js

```diff
@@ -17,6 +29,21 @@ describe('Index test', () => {
     expect(response.statusCode).toBe(302)
   })
 
+  test('GET / route returns 200 and clears previous details', async () => {
+    const options = {
+      method: 'GET',
+      url: '/',
+      auth
+    }
+
+    const response = await server.inject(options)
+    expect(response.statusCode).toBe(200)
```

Comment:
Being a bit fussy here, could we have a toBeCalledWith() on line 42 - 44?  Would more clearly prove the test case.  Alternatively we could have a clearCdo function that does it all.